src/ifcopenshell-python/ifcopenshell/geom/occ_utils.py

```diff
@@ -17,6 +17,10 @@
 #                                                                             #
 ###############################################################################
 
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Check line 185
https://github.com/IfcOpenShell/IfcOpenShell/pull/287/files#diff-a4812f80978da2638af11368d5da84b5R185

Reply:
You mean because of the usage of `map()` which returns a list / iterable in py2 / 3? I think it is fine in both versions, due to the `*` the elements get expanded into positional arguments to `gp_Pnt()` regardless.

Reply:
I was looking at that line, because as a test I ran the futurize script and this line was the only one where it detected a division. But I think that also here the new division is what really was intended, so it is ok as it is.

What really needs to be tested is whether any imports break on Python 2.

Reply:
Ah, I get it now, it's about the division and not about the map(). Both operands are floating point so `from __future__ import division` doesn't affect anything.